src/main/java/io/github/syst3ms/skriptparser/parsing/ParserState.java

```diff
@@ -4,39 +4,37 @@
 import io.github.syst3ms.skriptparser.lang.SyntaxElement;
 import io.github.syst3ms.skriptparser.lang.TriggerContext;
 
-import java.util.Collections;
-import java.util.LinkedList;
-import java.util.List;
+import java.util.*;
```

Comment:
i'm not sure that is really important to import all class of java.util, because this class use only three java.util class

Reply:
That's my IDE's fault. I'm not going to change this since it's really a nitpick.